backend/tests/test_views.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from django.test import Client
+
+client = Client()
+
+def test_api_root(client):
+    response = client.get('/api/v1/')
+    assert response.status_code == 200
+
+@pytest.mark.django_db
+def test_businesses_endpoint_list_response(client):
+    response = client.get('/api/v1/businesses/')
+    expected_keys =['camis', 'name', 'address', 'phone', 'cuisine_description']
+    assert response.status_code == 200
+    assert response.data['count'] == 20619
+    assert len(response.data['results']) == 100
+    for d in response.data['results']:
+        assert d.keys() == expected_keys
+
+@pytest.mark.django_db
+def test_businesses_endpoint_detail_response(client):
+    response = client.get('/api/v1/businesses/40394054/')
```

Comment:
@galarant i just hardcoded a camis here to test a detail response, but not sure what i should be doing...